collections-arrays/RemoveDuplicatesFromAnArray/Tests/Tests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using RemoveDuplicatesFromAnArray;
+using System.Linq;
+using Xunit;
+
+namespace Tests;
+
+public class Tests
```

Comment:
Again, since you're using the same array everywhere, you can declare it as a private field.